python/dgl/sparse/reduction.py

```diff
@@ -6,30 +8,28 @@
 from .sparse_matrix import SparseMatrix
 
 
-def reduce(A: SparseMatrix, dim: Optional[int] = None, rtype: str = "sum"):
-    """Compute the reduction of non-zero values in sparse matrix A along
-    the given dimension :attr:`dim`.
+def reduce(input: SparseMatrix, dim: Optional[int] = None, rtype: str = "sum"):
+    """Computes the reduction of non-zero values of the ``input`` sparse matrix
+    along the given dimension :attr:`dim`.
 
-    The reduction does not count zero values. If the row or column to be
-    reduced does not have any non-zero value, the result will be 0.
+    The reduction does not count zero elements. If the row or column to be
+    reduced does not have any non-zero elements, the result will be 0.
 
     Parameters
     ----------
-    A : SparseMatrix
```

Comment:
Currently we are using SparseMatrix.reduce = reduce, so the generated reduce function does not have the first parameter in the doc: `SparseMatrix.reduce(dim: Optional[int] = None, rtype: str = 'sum')`, so it is weird to have this in the doc about the first parameter.


https://docs.dgl.ai/en/latest/generated/dgl.sparse.SparseMatrix.reduce.html#dgl.sparse.SparseMatrix.reduce